packages/@sanity/default-layout/src/navbar/search/SearchField.tsx

```diff
@@ -1,111 +1,241 @@
-    )
+      if (!props.hidden && searchString && !loading && hits.length === 0) {
+        return (
+          <FullscreenContent>
+            <Flex align="center" height="fill" justify="center">
+              <Text align="center" muted>
+                No results for <strong>‘{searchString}’</strong>
+              </Text>
+            </Flex>
+          </FullscreenContent>
+        )
+      }
+
+      return (
+        <FullscreenContent hidden={props.hidden} ref={ref}>
+          {props.content}
+        </FullscreenContent>
+      )
+    },
+    [error, searchString, loading, hits, FullscreenContent]
+  )
+
+  const PopoverContent = useMemo(
+    () =>
+      // eslint-disable-next-line no-shadow
+      forwardRef(function LinkComponent(
+        popoverProps: PopoverProps,
+        ref: React.ForwardedRef<HTMLDivElement>
+      ) {
+        return (
+          <ResultsPopover
+            portal
+            placement="bottom"
+            arrow={false}
+            constrainSize
+            ref={ref}
+            scheme="light"
+            matchReferenceWidth
+            {...popoverProps}
+          />
+        )
+      }),
+    []
+  )
+
+  const renderPopover = useCallback(
+    (props, ref) => {
+      if (!props.hidden && error) {
+        return (
+          <PopoverContent
+            open={!props.hidden}
+            referenceElement={props.inputElement}
+            ref={ref}
+            content={
+              <Box padding={4}>
+                <PopoverContentFlex align="center" height="fill" justify="center">
+                  <Text align="center" muted>
+                    Something went wrong while searching
+                  </Text>
+                </PopoverContentFlex>
+              </Box>
+            }
+          />
+        )
+      }
+
+      if (!props.hidden && searchString && !loading && hits.length === 0) {
+        return (
+          <PopoverContent
+            open={!props.hidden}
+            referenceElement={props.inputElement}
+            ref={ref}
+            content={
+              <Box padding={4}>
+                <PopoverContentFlex align="center" height="fill" justify="center">
+                  <Text align="center" muted>
+                    No results for <strong>‘{searchString}’</strong>
+                  </Text>
+                </PopoverContentFlex>
+              </Box>
+            }
+          />
+        )
+      }
+
+      if (!props.hidden && hits.length > 0) {
+        return (
+          <PopoverContent
+            open={!props.hidden}
+            referenceElement={props.inputElement}
+            ref={ref}
+            content={props.content}
+          />
+        )
+      }
+
+      return undefined
+    },
+    [error, searchString, loading, hits, PopoverContent]
+  )
+
+  const autoComplete = useMemo(
+    () => (
+      <Autocomplete
+        icon={SearchIcon}
+        key="studio-search"
+        id="studio-search"
+        listBox={{padding: 2}}
```

Comment:
This can be removed, since we want `listBox={{padding: 1}}` (which is the default).

Reply:
Fixed!